src/parsing/parser.rs

```diff
@@ -6429,6 +6429,7 @@ fn allows_inline_multi_line(node: &Node, has_siblings: bool) -> bool {
         Node::AssignPat(node) => allows_inline_multi_line(&(&node.left).into(), has_siblings)
             || allows_inline_multi_line(&(&node.right).into(), has_siblings),
         Node::TsTypeAnn(type_ann) => allows_inline_multi_line(&(&type_ann.type_ann).into(), has_siblings),
+        Node::TsTupleElement(tuple_element) => allows_inline_multi_line(&(&tuple_element.ty).into(), has_siblings),
```

Comment:
There is this "inline multi-line" functionality that allows for stuff like:

```ts
call({
  obj: "prop",
});
```

Instead of:

```ts
call(
  {
    obj: "prop",
  }
);
```

There's now a new `TsTupleElement` node inbetween where it previously checked so it just has to check the type of that now. (As you probably know `TsTupleElement` was added in swc to support the new labeled tuple elements).

Reply:
Ah right, somehow I missed that. Thanks for explanation!